src/components/EnterpriseSubsidiesContext/index.jsx

```diff
@@ -0,0 +1,20 @@
+import { createContext, useMemo } from 'react';
+import { useEnterpriseOffers } from '../enterprise-offers/data/hooks';
```

Comment:
nit/suggestion: not blocking, but curious about having this hook live in `components/enterprise-offers`, if `enterprise-offers` folder doesn't actually contain any components? Would it make sense to co-locate this hook where it's called in `components/EnterpriseSubsidiesContext`?